lib/fluid/model/resources/pool.ex

```diff
@@ -2,7 +2,8 @@ defmodule Fluid.Model.Pool do
   require Logger
 
   use Ash.Resource,
-    data_layer: AshPostgres.DataLayer
+    data_layer: AshPostgres.DataLayer,
+    extensions: [AshJason.Extension]
```

Comment:
Why use Jason now? What's the thinking here?

Reply:
we are serialising the entire struct as map to store in db. currently, this is used in `Tag` module to store `source` and `destination`. 

storing data in this de-normalised way avoid excessive db calls. This way, `Tag` has `source` and `destination` whose data is stored as map and can be accesssed directly.